docker-images/jenkins/Dockerfile

```diff
@@ -50,3 +43,7 @@ RUN unzip /wso2am-micro-gw-toolkit-3.0.1.zip && \
 rm /wso2am-micro-gw-toolkit-3.0.1.zip
 
 ENV PATH="${PATH}:/wso2am-micro-gw-toolkit-3.0.1/bin:/usr/local/apache-jmeter-5.1/bin"
+
+#Pre-Install Jenkins Plugins
+COPY plugins.txt /usr/share/jenkins/ref/plugins.txt
+RUN /usr/local/bin/install-plugins.sh < /usr/share/jenkins/ref/plugins.txt
```

Comment:
@Aaquiff there is no need to delete the Micro Gateway pack if we are copying it (using the `COPY` instruction) [1]. The pack will anyway exist in one of the image layers.

[1]: [Code segment](https://github.com/wso2/kubernetes-pipeline/blob/dbb444a1191c47f27905e0821c9d7b475cc7d9bd/docker-images/jenkins/Dockerfile#L41-L43)

Reply:
Removed the command to delete the pack